### PR TITLE
refactor: NEAR tokens usages to use a strictly typed near-token crate

### PR DESCRIPTION
**Pull request by FroVolod**

### CODE REVIEW

File: src/common.rs [hunk 1]
```diff
@@ -139,19 +137,19 @@ impl std::fmt::Display for AccountTransferAllowance {
 }
 
 impl AccountTransferAllowance {
-    pub fn liquid_storage_stake(&self) -> NearBalance {
-        NearBalance::from_yoctonear(
+    pub fn liquid_storage_stake(&self) -> near_token::NearToken {
+        near_token::NearToken::from_yoctonear(
```
Comment (frol) [inline src/common.rs:141]: No need to reconstruct NearToken from yoctonearto perform math operations on NearToken, there are dedicated methods implemented:

https://docs.rs/near-token/latest/near_token/struct.NearToken.html#method.saturating_sub

File: src/common.rs [hunk 2]
```diff
@@ -187,15 +185,15 @@ pub fn get_account_transfer_allowance(
...
+        account_liquid_balance: near_token::NearToken::from_yoctonear(account_view.amount),
+        account_locked_balance: near_token::NearToken::from_yoctonear(account_view.locked),
+        storage_stake: near_token::NearToken::from_yoctonear(
             u128::from(account_view.storage_usage) * storage_amount_per_byte,
         ),
         // pessimistic_transaction_fee = 10^21 - this value is set temporarily
         // In the future, its value will be calculated by the function: fn tx_cost(...)
         // https://github.com/near/nearcore/blob/8a377fda0b4ce319385c463f1ae46e4b0b29dcd9/runtime/runtime/src/config.rs#L178-L232
-        pessimistic_transaction_fee: NearBalance::from_yoctonear(10u128.pow(21)),
+        pessimistic_transaction_fee: near_token::NearToken::from_yoctonear(10u128.pow(21)),
```
Comment (frol) [inline src/common.rs:196]: Use from_millinear(1)

**Review by frol — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -74,12 +74,18 @@ near-gas = { version = "0.2.3", features = [
     "schemars",
     "interactive-clap",
 ] }
+near-token = { version = "0.2.0", features = [
+    "serde",
+    "borsh",
+    "schemars",
```
[the remark this answers is not in the document]
Comment (FroVolod) [reply Cargo.toml:80]: Done

File: src/common.rs (hunk 1 above)
Comment (FroVolod) [reply src/common.rs]: Done

**Review by frol — APPROVED** [review]

**Pull request merged**